types/index.d.ts

```diff
@@ -162,6 +162,7 @@ export interface Column<RowData extends object> {
     data2: RowData,
     type: 'row' | 'group'
   ) => number;
+  customExport?: (rowData: RowData) => void;
```

Comment:
This return value should not be void, but instead unknown, so anything could be returned.